website/client/js/services/userServices.js

```diff
@@ -310,6 +310,19 @@ angular.module('habitrpg')
           });
         },
 
+        setCustomDayStart: function (dayStart) {
+          $http({
+            method: "POST",
+            url: 'api/v3/user/set-custom-day-start',
+            data: {
+              dayStart: dayStart
+            }
+          })
+          .then(function (response) {
+            Notification.text('Day start updated. Remember to refresh');
```

Comment:
This function will set the last cron as well according to Paglias note above. So, it will do the same thing as the debug and some extra.